.github/workflows/release-chart.yaml

```diff
@@ -34,6 +34,8 @@ jobs:
         run: |
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      - name: Add newrelic repository
+        run: helm repo add newrelic https://helm-charts.newrelic.com
```

Comment:
solve this issue https://github.com/newrelic/nri-prometheus/runs/6090392009?check_suite_focus=true